pkg/services/sqlstore/migrations/ualert/migration_test.go

```diff
@@ -0,0 +1,548 @@
+package ualert_test
+
+import (
```

Comment:
I suggest you move all of the helper / private methods to the bottom of the file and only leave at the top the actual tests.

It's consistent with what we do and follows the common (i think) way of reading - tests first, helper functions later - which you can typically infer what they do with their names.

Reply:
I think it makes sense if the usage of those helper methods is across many files. In this case, this is the single place where utils are used. What do you think about alternative: move helper methods at the end of the file?